backend/src/test/scala/cromwell/backend/wdl/PureFunctionsSpec.scala

```diff
@@ -23,7 +24,7 @@ class PureFunctionsSpec extends FlatSpec with Matchers {
       WdlArray(WdlArrayType(WdlIntegerType), List(WdlInteger(3), WdlInteger(6)))
     ))
 
-    OnlyPureFunctions.transpose(Seq(Success(inArray))) should be(Success(expectedResult))
+    PureStandardLibraryFunctions.transpose(Seq(Success(inArray))) should be(Success(expectedResult))
```

Comment:
I'll rename this here. Since they really belong in WDL4s I've made a note to move them across (and add the new ones) in https://github.com/broadinstitute/wdl4s/pull/43